Scripts/RemediationScripts/Remediate-EnableEncryptionInTransitForStorageAccounts.ps1

```diff
@@ -0,0 +1,691 @@
+            }        
+            else
+            { 
+                 $stgWithDisableHTTPS += $_|Select-Object  @{N='ResourceId';E={$_.Id}},
+                                                           @{N='ResourceGroupName';E={$_.ResourceGroupName}},
+                                                           @{N='StorageAccountName';E={$_.StorageAccountName}},
+                                                           @{N='EnableHttpsTrafficOnly';E={$_.EnableHttpsTrafficOnly}}                                                                  
+            }
+        }                    
+                              
+        $totalStgWithEnableHTTPS = ($stgWithEnableHTTPS | Measure-Object).Count
+        $totalStgWithDisableHTTPS = ($stgWithDisableHTTPS | Measure-Object).Count
+    
+        Write-Host "Storage Account(s) with enabled 'secure transfer': [$($totalStgWithEnableHTTPS)]" -ForegroundColor $([Constants]::MessageType.Update)
+        Write-Host "Storage Account(s) with disabled 'secure transfer'': [$($totalStgWithDisableHTTPS)]" -ForegroundColor $([Constants]::MessageType.Update)
+
+        #Start remediation Storage Account(s) with 'secure transfer' enabled.
+        if ($totalStgWithDisableHTTPS -gt 0)
+        {
+            # Creating the log file
+            if (-not $DryRun)
+            {  
+                write-host  $([Constants]::DoubleDashLine)
+                Write-Host "[step 5 of 6] : Exporting the Storage Account to csv file so that it can be used to rollback."
+                Write-Host "Backing up config of Storage Account(s) details. Please do not delete this file. Without this file you won't be able to rollback any changes done through remediation script." 
+                $stgWithDisableHTTPS | Export-CSV -Path "$($folderpath)\StorageWithDisableHTTPS.csv" -NoTypeInformation
+                Write-Host "Path: $($folderPath)\StorageWithDisableHTTPS.csv" -ForegroundColor $([Constants]::MessageType.Update)
+                write-host $([Constants]::SingleDashLine)
+               
+                if (-not $Force)
+                {
+                    Write-Host "Do you want to enable secure transfer on the storage account(s)? " -ForegroundColor $([Constants]::MessageType.Warning) -NoNewline
+                    $userInput = Read-Host -Prompt "(Y|N)"
+    
+                    if($userInput -ne "Y")
+                    {
+                        Write-Host "Secure transfer will not be enabled for any of the storage account. Exiting..." -ForegroundColor $([Constants]::MessageType.Info)
+                        break
+                    }
+                }
+                else
+                {
+                    Write-Host "'Force' flag is provided. Secure transfer will be enabled on the storage account without any further prompts." -ForegroundColor $([Constants]::MessageType.Warning)
+                }
+   
+                write-host $([Constants]::DoubleDashLine)
```

Comment:
`write-host` to be replaced with `Write-Host`.

Reply:
done